accounts/urls.py

```diff
@@ -1,10 +1,15 @@
-from django.conf.urls import patterns, url
+from django.conf.urls import patterns, url, include
 from django.views.generic import TemplateView
+#from social_auth.views import
 
 from accounts import views
 
 urlpatterns = patterns('',
-    url(r'^$', views.NewUserView.as_view(), name='accounts'),
+    url(r'^$', views.NewUserView.as_view(), name='register'),
     url(r'^success/$', TemplateView.as_view(template_name="accounts/success.html"), name='success_created'),
-    url(r'^activate/(?P<activation_key>\w+)/$', views.activate)
+    url(r'^activate/(?P<activation_key>\w+)/$', views.activate),
+    url('', include('social.apps.django_app.urls', namespace='social')),
```

Comment:
Czyli załaduje ustawienia z settings ```SOCIAL_AUTH_LOGIN_URL``` i będzie próbował się dostać do url'a ```/login/```, który również jest poniżej czyż nie? Jeżeli tak to, które z nich zostanie pierwszy obsłużony? Nie uważasz, że lepiej dać tutaj np. ```'social/'```?